### PR TITLE
data:hussain-shariff

### PR DESCRIPTION
**Pull request by hussain-shariff**
## Adding my profile to Linktree

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2849"></a>

### CODE REVIEW

File: hussain-shariff.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Hussain Shariff",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Mechanical engineer exploring web dev | Starting with Open Source journey",
+  "tags": ["reactJs", "nextJs", "tailwindCSS", "Sanity"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/hussainshariff_" },
+    { "icon": "FaGithub", "url": "https://github.com/hussain-shariff"},
```
Comment (Njong392) [inline hussain-shariff.json:9]: I believe there is a trailing comma here.
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/hussain-shariff"}
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for contributing :tada: Please take a look at the changes requested.

File: hussain-shariff.json (hunk 1 above)
Comment (eddiejaoude) [reply hussain-shariff.json:9]: Good spot @Njong392 👍

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please move your profile to the data directory.

File: hussain-shariff.json (hunk 1 above)
Comment (hussain-shariff) [reply hussain-shariff.json]: @Njong392 how do I make changes now?
Should I create a new json file and create PR or there is some way I can edit this?

**Comment by hussain-shariff** [comment]
> Please move your profile to the data directory.

Could you please tell me how to do this?

**Comment by dan-mba** [comment]
> > Please move your profile to the data directory.
> 
> Could you please tell me how to do this?

If you are working locally on a PC or Mac just use normal file movement commands or File Explorer / Finder.
If you are working in the GitHub editor, when you edit a file there is a box at the top to edit the name.
Just put `data/` in front of your filename and it should move to the data directory.

**Review by dan-mba — APPROVED** [review]
Looks good to me

File: hussain-shariff.json (hunk 1 above)
Comment (hussain-shariff) [reply hussain-shariff.json]: > @Njong392 how do I make changes now? Should I create a new json file and create PR or there is some way I can edit this?

Nevermind I made the changes.
Is it okay now?

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 🎉 
Great collaboration Everyone! 👍

**Pull request merged**